packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -86,6 +86,53 @@ All email subjects, bodies and SMS messages for both invitation and verification
 Learn more about [message templates
 here](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-settings-message-templates.html).
 
+### Sign In
+
+Users registering or signing in into your application can do so with multiple identifiers. There are 4 options
+available:
+
+* `USERNAME`: Allow signing in using the one time immutable user name that the user chose at the time of sign up.
+* `EMAIL`: Allow signing in using the email address that is associated with the account.
+* `PHONE_NUMBER`: Allow signing in using the phone number that is associated with the account.
+* `PREFERRED_USERNAME`: Allow signing in with an alternate user name that the user can change at any time. However, this
+  is not available if the USERNAME option is not chosen.
+
+The following code sets up a user pool so that the user can sign in with either their username or their email address -
+
+```ts
+new UserPool(this, 'myuserpool', {
+  // ...
+  // ...
+  signInAliases: [ SignInAlias.USERNAME, SignInAlias.EMAIL ],
```

Comment:
Perhaps a set of boolean flags is more discoverable and suitable for this configuration:

```ts
signInAliases: {
  userName: true,
  email: true,
  // ...
}
```